rpcs3/Emu/SysCalls/Modules/cellSync.cpp

```diff
@@ -1596,6 +1596,41 @@ s32 _cellSyncLFQueueDetachLv2EventQueue(vm::ptr<u32> spus, u32 num, vm::ptr<Cell
 
 Module cellSync("cellSync", []()
 {
+	// setup error handler
+	cellSync.on_error = [](s64 value, ModuleFunc* func)
+	{
+		// get error name for CELL_SYNC errors
+		auto get_error = [](s32 code) -> const char*
```

Comment:
Are you sure that here and down below should be s32 and not u32?
Those enum cases cannot be narrowed down to s32. (higher than 2^32)